client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/navigator-header/style.scss

```diff
@@ -38,3 +38,13 @@
 		letter-spacing: -0.15px;
 	}
 }
+
+.rtl {
+	.pattern-assembler {
+		.navigator-header__heading {
+			.gridicon {
+				transform: rotate(180deg);
```

Comment:
I saw we have handled RTL for gridicon. Maybe it should work and we don't need this one 🤔

https://github.com/Automattic/wp-calypso/blob/a53396c3a683bdf92c21cb91ca0e676d23ca36e2/client/assets/stylesheets/_main.scss#L117

Reply:
I could not find the style (or anything) for RTL for this, and I see it's not working.
<img width="501" alt="Screen Shot 2023-03-31 at 15 43 18" src="https://user-images.githubusercontent.com/5287479/229043796-2bcd40d7-b1c7-4b23-8235-f1356e34dff9.png">

I'd appreciate it if you let me know where that is implemented!

Reply:
Looks like the entry point of the `/setup` path doesn't import that styles. We can simply import it and see whether any side effect 😆

```js
// client/landing/stepper/index.tsx
import 'calypso/assets/stylesheets/style.scss';
```

Reply:
Thanks, I fixed it with this commit.
https://github.com/Automattic/wp-calypso/pull/75112/commits/1afe45179cfa7ac98bcdb30dd4d739cb4589f019

I only found the side effect that `iframe` shrinks, as below. So added the style overriding it. 👨🏻‍🔬
<img width="700" alt="Screen Shot 2023-03-31 at 17 03 05" src="https://user-images.githubusercontent.com/5287479/229061962-8df82359-f798-49d7-a156-5acd0be4eaf5.png">
<img width="344" alt="Screen Shot 2023-03-31 at 17 03 17" src="https://user-images.githubusercontent.com/5287479/229062020-6aa20c9b-30e7-401b-8b8c-55b8158b180c.png">